### PR TITLE
eks template

### PR DESCRIPTION
**Pull request by joshuafernandes**

### CODE REVIEW

File: ibft2-eks/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+5. Deploy the Besu chart
+
+Clone the example [besu kubernetes repo](https://github.com/PegaSysEng/besu-kubernetes) and select one of the helm deployments.
+
+Lets say 'ibft2' so to deploy the chart:
+
+```
+cd helm\ibft2
+KUBECONFIG=$KUBECONFIG:/home/<username>/workspace/besu-aws/ibft2-k8s/.kube/config helm install besu ./besu
```
Comment (EdJoJob) [inline ibft2-eks/README.md]: can we move this path reference to somethinh like `/path/to/repo/.kube/config`

Comment (EdJoJob) [reply ibft2-eks/README.md]: and everywhere else we refer to the `/home/<username>/workspace/besu-aws/` path

File: ibft2-eks/cfn-besu-ibft2-eks.yml [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+    Type: "AWS::EC2::VPC::Id"
+    Description: The ID of your existing VPC that we will deploy into (e.g., vpc-012345)
+
+  PublicSubnetAId:
+    Default: "subnet-"
+    Description: The ID of the public subnet in AZ 1
+    Type: String
+
+  PublicSubnetBId:
+    Default: "subnet-05990c77cc8bf62a8"
```
Comment (EdJoJob) [inline ibft2-eks/cfn-besu-ibft2-eks.yml]: I think this default needs to be changed to be a fake subnet ID

File: ibft2-eks/cfn-besu-ibft2-eks.yml [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+      IpProtocol: tcp
+      FromPort: 443
+      ToPort: 443
+
+  NodeSSHSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: node ssh sg
+      SecurityGroupEgress:
+        - CidrIp: 0.0.0.0/0
```
Comment (EdJoJob) [inline ibft2-eks/cfn-besu-ibft2-eks.yml]: We should ref the `SSHCidrRange` param

File: ibft2-eks/.kube/config.template [hunk 4]
```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+clusters:
+- cluster:
+    server: <INSERT THE CLUSTER ENDPOINT FROM THE CFN OUTPUTS>
+    certificate-authority-data: <INSERT THE CA DATA FROM THE EKS CONSOLE>
```
Comment (EdJoJob) [inline ibft2-eks/.kube/config.template]: Would it be possible or desirable to have a script to get these values using the AWS-CLI?

File: ibft2-eks/aws-auth-cm.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: aws-auth
+  namespace: kube-system
+data:
+  mapRoles: |
+    - rolearn: <INSERT ARN HERE FROM CFN OUTPUTS>
```
Comment (EdJoJob) [inline ibft2-eks/aws-auth-cm.yml]: Would it be possible or desirable to have a script to get this value using the AWS-CLI?

File: ibft2-eks/cfn-besu-ibft2-eks.yml (hunk 2 above)
Comment (joshuafernandes) [reply ibft2-eks/cfn-besu-ibft2-eks.yml]: done

File: ibft2-eks/README.md (hunk 1 above)
Comment (joshuafernandes) [reply ibft2-eks/README.md]: done

File: ibft2-eks/.kube/config.template (hunk 4 above)
Comment (joshuafernandes) [reply ibft2-eks/.kube/config.template]: done, using a script now and have updated the readme

**Review by EdJoJob — APPROVED** [review]

**Pull request merged**